datadog/resource_datadog_service_level_objective_test.go

```diff
@@ -101,6 +101,58 @@ resource "datadog_service_level_objective" "foo" {
 }`, uniq)
 }
 
+func testAccCheckDatadogServiceLevelObjectiveConfigMonitor(uniq string) string {
+	return fmt.Sprintf(`
+resource "datadog_monitor" "foo" {
+  name = "Name for monitor foo"
```

Comment:
please make sure that _all_ resource names contain `%s` with `uniq` so we can identify these resources in test org

Reply:
Oh, thanks, I missed this point